Sig.App.Frontend/src/views/beneficiary/AssignSubscriptions.vue

```diff
@@ -692,10 +692,41 @@ const amountThatWillBeAllocatedModalMoneyFormat = computed(() => {
   return amount !== 0 ? getMoneyFormat(amount) : "0$";
 });
 
+const { result: resultForecastAddingFundTransactionForSubscriptionByBeneficiary } = useQuery(
+  gql`
+    query ForecastAddingFundTransactionForSubscriptionByBeneficiary($subscriptionId: ID!, $beneficiaryIds: [ID!]) {
+      forecastAddingFundTransactionForSubscriptionByBeneficiary(
+        subscriptionId: $subscriptionId
+        beneficiaryIds: $beneficiaryIds
+      ) {
+        beneficiaries {
+          beneficiaryId
+          count
+        }
+      }
+    }
+  `,
+  ForecastAddingFundTransactionForSubscriptionByBeneficiaryVariables,
+  () => ({
+    enabled: selectedSubscription.value !== null && selectedBeneficiaries.value.length > 0
+  })
+);
+
+function ForecastAddingFundTransactionForSubscriptionByBeneficiaryVariables() {
+  return {
+    subscriptionId: selectedSubscription.value,
+    beneficiaryIds: selectedBeneficiaries.value.map((x) => x.id)
+  };
+}
```

Comment:
Quel est l'intérêt d'une fonction à part? Et le nom qui commence par une lettre majuscule m'a confondu...

```suggestion
  () => ({
    subscriptionId: selectedSubscription.value,
    beneficiaryIds: selectedBeneficiaries.value.map((x) => x.id)
  }),
  () => ({
    enabled: selectedSubscription.value !== null && selectedBeneficiaries.value.length > 0
  })
);

function ForecastAddingFundTransactionForSubscriptionByBeneficiaryVariables() {
  return {
    subscriptionId: selectedSubscription.value,
    beneficiaryIds: selectedBeneficiaries.value.map((x) => x.id)
  };
}
```